sormas-api/src/main/resources/strings.properties

```diff
@@ -363,6 +363,7 @@ errorSormasToSormasShareContactWithUnsharedSourceCase = If you want to share thi
 errorSormasToSormasAcceptContactsWithoutCaseShared = The contact <b>%s</b> cannot be accepted, because it was sent without a case. Please reject the contact and ask the source system to send the linked case first.
 errorSormasToSormasAcceptContactsBeforeAcceptSourceCase = The contact <b>%s</b> cannot be accepted, because the contact belongs to a case that has not yet been accepted. Please accept the linked case first.<br/>It is shared within the request <b>%s</b>.
 errorSormasToSormasExistingPendingRequest = There is already a pending request to the same organization. Please revoke it before sending a new one.
+errorSormasToSormasSharequestNotFound = The share request could not be found. It was revoked by the source system or it has been rejected by somone else.
```

Comment:
Please change text to: The share request could not be found. It was either revoked by the source system or it has been rejected by somone else.